include/itkTorchImageToImageFilter.h

```diff
@@ -19,60 +19,88 @@
 #define itkTorchImageToImageFilter_h
 
 #include "itkImageToImageFilter.h"
+#include "itkImageRegionIterator.h"
+#include "itkImageRegionConstIterator.h"
+
+#include "itkTorchImage.h"
 
 namespace itk
 {
 
 /** \class TorchImageToImageFilter
  *
- * \brief Filters a image by iterating over its pixels.
+ * \brief Converts an itk::TorchImage to an itk::Image.
  *
- * Filters a image by iterating over its pixels in a multi-threaded way
- * and {to be completed by the developer}.
+ * Conversions between TorchImage and Image permit pipelines with
+ * steps for either kind of image.
  *
  * \ingroup PyTorch
  *
  */
-template <typename TInputImage, typename TOutputImage>
-class TorchImageToImageFilter : public ImageToImageFilter<TInputImage, TOutputImage>
+template< typename TInputTorchImage, typename TOutputImage >
+class ITK_TEMPLATE_EXPORT TorchImageToImageFilter : public ImageToImageFilter< TInputTorchImage, TOutputImage >
 {
 public:
-  ITK_DISALLOW_COPY_AND_ASSIGN(TorchImageToImageFilter);
+  static constexpr bool ConceptCheck =
+    std::is_same< TInputTorchImage, TorchImage< typename TInputTorchImage::PixelType, TInputTorchImage::ImageDimension > >::value &&
+    std::is_same< TOutputImage, Image< typename TOutputImage::PixelType, TOutputImage::ImageDimension > >::value &&
+    std::is_same< typename TInputTorchImage::PixelType, typename TOutputImage::PixelType >::value &&
```

Comment:
lines 45 and 46 are redundant given lines 47 and 48.

Reply:
Will do.